stock_reception_screen/models/stock_reception_screen.py

```diff
@@ -532,6 +532,13 @@ def _check_package_data(self):
             msg = _("The storage type is mandatory before going further.")
             self.env.user.notify_warning(message="", title=msg)
             return False
+        if self.current_move_line_product_packaging_id and (
+            not self.current_move_line_product_packaging_id.volume
```

Comment:
The volume is in m3 and when the packaging is small, it equals to 0 after roundnig. Can you check instead the length/width/height that are in mm ?